ubuntu_image/hooks.py

```diff
@@ -0,0 +1,60 @@
+"""A general hook mechanism"""
```

Comment:
I would add more documentation here.

Something like:

"""A general hook mechanism.

This module defines a HookManager that will run all the executable scripts found in the passed-in hooks directory(s) once the rootfs was created.

Each of the scripts will get the location of the rootfs as the "UBUNTU_IMAGE_HOOK_ROOTFS" environment variable."""